feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts

```diff
@@ -7,28 +14,39 @@ import {
   ProductImportStatus,
   ProductImportSummary,
 } from '@spartacus/cart/base/root';
+
 import {
   FocusConfig,
   ICON_TYPE,
   LaunchDialogService,
 } from '@spartacus/storefront';
+
 import { BehaviorSubject, Observable } from 'rxjs';
 import { finalize, pluck } from 'rxjs/operators';
-
 @Component({
   selector: 'cx-import-entries-dialog',
   templateUrl: './import-entries-dialog.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ImportEntriesDialogComponent {
+export class ImportEntriesDialogComponent implements OnDestroy{
   iconTypes = ICON_TYPE;
   focusConfig: FocusConfig = {
     trap: true,
     block: true,
-    autofocus: 'button',
-    focusOnEscape: true,
+    autofocus: false,
+    focusOnEscape: false,
   };
 
+ @HostListener('click', ['$event'])
+  handleClick(event: UIEvent): void { 
+    console.log("target name =" + (event.target as any).tagName);
```

Comment:
Should these console logs statements be there?

Reply:
Removed.